package.json

```diff
@@ -42,7 +42,7 @@
     "lodash.isempty": "^4.4.0",
     "lodash.sortby": "^4.7.0",
     "micromatch": "4.0.7",
-    "snyk-nodejs-lockfile-parser": "^1.56.1",
+    "snyk-nodejs-lockfile-parser": "file:../snyk-nodejs-lockfile-parser",
```

Comment:
don't forget to add the actual version here

Reply:
yup, changed it